app/views/components/cookie-banner/index.html

```diff
@@ -0,0 +1,37 @@
+{% extends "layouts/base.html" %}
+
+{% block body %}
+
+  <div class="govuk-grid-row">
+
+    <div class="govuk-grid-column-full">
+
+      {{ govukBackLink({
+        text: "Back",
+        href: "../"
+      }) }}
+
+      <h1 class="govuk-heading-xl">
+        <span class="govuk-caption-xl">Components</span> Cookie banner
+      </h1>
+
+      {{ mojCookieBanner({
+        heading: {
+          title: 'Tell us whether you accept cookies',
+          html: 'GOV.UK uses cookies which are essential for the site to work. We also use non-essential cookies to help us improve government digital services. Any data collected is anonymised.'
+        },
+        acceptCookies: {
+          type: 'submit',
+          text: 'Accept all cookies'
+        },
+        setCookies: {
+          href: '/cookies',
+          text: 'Set application cookies'
+        }
```

Comment:
I think these arguments are too limiting: you can only use two buttons with pre-defined purposes, one is a button the other a link. I'd rather flexibility (so you can add "reject all", or give them both JS events) and a good copy-pastable example.

I think I'd prefer if this was an array of button configurations that are passed to GDS's [button component](https://design-system.service.gov.uk/components/button/). e.g.:

```suggestion
        [
            {
                value: 'Accept all cookies',
            },
            {
                href: '/cookies',
                text: 'Set application cookies'
            }
        ]
```

Reply:
Yeah I like this idea, a lot more flexible, I will add this in

Reply:
So in there we need the option of rendering a `button` or a `a`. In nunjucks, is it possible to specify nicely one or the other? Would you pass in a type property and render the appropriate element?

Reply:
If you're using the GDS button component, you can pass in `element`. But it also automatically infers it from whether `value`(button) or `text`/`html`(a) was provided.([Button Nunjucks options](https://design-system.service.gov.uk/components/button/#options-button-example))